lib/rollbax/client.ex

```diff
@@ -109,6 +114,10 @@ defmodule Rollbax.Client do
     {:noreply, new_state}
   end
 
+  def handle_info(:lift_rate_limit, state) do
```

Comment:
Lift in case of rate limiting feels that that we actually increasing the rate limiting threshold.
How about `cease_rate_limiting` (alternatively, `stop_rate_limiting`)?

Reply:
I think the correct term is "lifting" the limit, so I'm with @qcam here.

Reply:
I wonder, do you have an example where "lift the limit" is used not in context of increasing the limit number itself? 🤔

Reply:
@lexmag https://www.merriam-webster.com/dictionary/lift, one definition is "REVOKE, RESCIND" which is what we want in this case? I'm fine with `stop_rate_limiting` too.